ksp_plugin/plugin.hpp

```diff
@@ -122,6 +122,15 @@ class Plugin {
       serialization::GravityModel::Body const& gravity_model,
       serialization::InitialState::Keplerian::Body const& initial_state);
 
+  virtual void SetEphemerisParameters(
```

Comment:
The word "set" is a bit of a misnomer since it seems to imply that this can be set at any time, while it's only at initialization.  I would call this `InitializeEphemerisParameters`.  Same for the history and psychohistory and for the interface methods.

Reply:
Done.